### PR TITLE
New feature: CLM filter templates

### PR DESCRIPTION
**Pull request by cbbayburt**
# Description

New feature: CLM filter templates

# Target branches

- [x] Master (Default)
- [x] Manager-4.2

# Links

Feature PR: https://github.com/uyuni-project/uyuni/pull/3561

### CODE REVIEW

File: modules/administration/pages/content-lifecycle-examples.adoc
Comment (keichwa) [inline modules/administration/pages/content-lifecycle-examples.adoc:187]: Maybe, we can turn this into a formal note (it's new, thus it may deserve more visibility):
```suggestion
[NOTE]
====
This procedure can be automated using a filter template.
For more information on how to apply a live patching filter template, see xref:administration:content-lifecycle.adoc#filter-templates[].
====
```

File: modules/administration/pages/content-lifecycle.adoc
Comment (keichwa) [inline modules/administration/pages/content-lifecycle.adoc:120]:
```suggestion
When applied, these templates help creating a set of filters in advance, tailored for a specific use case.
```

File: modules/administration/pages/content-lifecycle.adoc
Comment (keichwa) [inline modules/administration/pages/content-lifecycle.adoc:132]: I think asciidoc wants two tics...
```suggestion
* Deny patches that contain ``kernel-default`` package greater than a specific kernel version
```

File: modules/administration/pages/content-lifecycle.adoc
Comment (keichwa) [inline modules/administration/pages/content-lifecycle.adoc:133]:
```suggestion
* Deny patches that contain ``reboot_suggested`` keyword
```

File: modules/administration/pages/content-lifecycle.adoc
Comment (keichwa) [inline modules/administration/pages/content-lifecycle.adoc:137]: We have this strange policy (I think it help adoc editors): New sentence = new line
```suggestion
The second "catch-all" filter is a safeguard to make sure the assigned clients are never requiring a restart in the future.
You can choose not to attach it to the project if this behavior is not desired because it might block other important updates.
```

File: modules/administration/pages/content-lifecycle.adoc
Comment (keichwa) [inline modules/administration/pages/content-lifecycle.adoc:152]:
```suggestion
. Select the two filters that have the specified prefix, and click btn:[Save].
```

**Review by keichwa — APPROVED** [review]
Thank your Can! Some minor suggestions.

@0rnela , can you help with English phrases? I'm not sure about my proposals!

**Comment by cbbayburt** [comment]
Thanks for the quick review @keichwa ! I've addressed your comments in a separate commit. Let's wait for @0rnela 's approval and then we can merge it.

**Review by 0rnela — APPROVED** [review]

**Pull request merged**
